src/models/Settings.js

```diff
@@ -79,6 +79,7 @@ function (Okta, Q, Errors, BrowserFeatures, Util, Logger, OAuth2Util, config) {
       'features.hideSignOutLinkInMFA' : ['boolean', false, false],
       'features.customExpiredPassword': ['boolean', true, false],
       'features.registration': ['boolean', false, false],
+      'features.emailRecovery': ['boolean', false, true],
```

Comment:
nit: can you move this next to the other recovery factors (about 9 lines above)?
Also 👍 for setting it to ```true``` by default, and not required, that will make it backwards compatible